py_cui/debug.py

```diff
@@ -310,12 +310,13 @@ def _get_debug_text(self, text):
         msg : str
             Log message with function, file, and line num info
         """
-
-        func = inspect.currentframe().f_back.f_back.f_code
+        current_frame  = inspect.currentframe()
+        if current_frame and current_frame.f_back and current_frame.f_back.f_back  is not None:
+            func = current_frame.f_back.f_back.f_code
```

Comment:
I'm not sure if checking for None here makes sense, since if we don't get a value for func, the return statement would fail anyway, no?

Reply:
I have done this to respect mypy. Else it complains about `Type of f_back`. Let me know if I should remove it!